packages/klevu-ui/src/components/klevu-rating/klevu-rating.tsx

```diff
@@ -0,0 +1,37 @@
+import { Component, Host, Prop, h } from "@stencil/core"
+
+/**
+ * Klevu ratings component
+ *
+ * @prop rating - The rating value as a number to show
+ */
+@Component({
+  tag: "klevu-rating",
+  styleUrl: "klevu-rating.css",
+  shadow: true,
+})
+export class KlevuRating {
+  /**
+   * Rating value
+   */
+  @Prop() rating: number = 0
+
```

Comment:
Should we create another prop to indicate what is the total rating stars instead of hard coded 5?

Reply:
added prop for ratingRange